packages/flame/lib/src/components/isometric_tile_map_component.dart

```diff
@@ -101,6 +104,11 @@ class IsometricTileMapComponent extends PositionComponent {
     }
   }
 
+  @override
+  void update(double dt) {
+    _recomputeSize();
```

Comment:
since the block matrix is not final, theoretically it could be changed at any time. luckily recomputing the size is quite cheap so I think this should be fine.